### PR TITLE
chore: remove some dumplicated code in ops.lua

### PR DESCRIPTION
**Pull request by YuanYingdong**
### What this PR does / why we need it:
When reading apisix code, I found that there are some duplicated codes in `ops.lua` (see the changes) and want to make the code cleaner.

### Pre-submission checklist:

* [X] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [X] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: apisix/cli/ops.lua [hunk 1]
```diff
@@ -282,77 +282,51 @@ Please modify "admin_key" in conf/config.yaml .
...
-                ip = yaml_conf.apisix.admin_listen.ip or ip
-                port = tonumber(yaml_conf.apisix.admin_listen.port) or port
-            end
-
-            if ports_to_check[port] ~= nil then
-                util.die("admin port ", port, " conflicts with ", ports_to_check[port], "\n")
+                admin_server_addr = validate_and_get_listen_addr("admin port", "0.0.0.0",
+                                            yaml_conf.apisix.admin_listen.ip,
+                                            9180, yaml_conf.apisix.admin_listen.port)
+            else
```
Comment (spacewander) [inline apisix/cli/ops.lua:304]: we can remove the outer check `if yaml_conf.apisix.admin_listen or yaml_conf.apisix.port_admin then` now?

Comment (YuanYingdong) [reply apisix/cli/ops.lua:304]: The old code indicates that `yaml_conf.apisix.admin_listen` and  `yaml_conf.apisix.port_admin` may be both `nil`, I think we need to keep the code cosistent.

Comment (spacewander) [reply apisix/cli/ops.lua:304]: Yes, we can change `else` to `elseif yaml_conf.apisix.port_admin`?

Comment (YuanYingdong) [reply apisix/cli/ops.lua]: Updated

**Review by spacewander — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**